### PR TITLE
Review your project from trials!

### PR DESCRIPTION
**Pull request by saruuja**
This project means how to perform a code review on my code!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,7 +44,7 @@ <h3>Here's a al-Khwarizmi's life and Contributions:</h3>
           based on very little evidence. In [1] Toomer suggests that the name
           al-Khwarizmi may indicate that he came from Khwarizm south of the
           Aral Sea in central Asia. He then writes:-."
-      
+        
       </ul>
     </div>
```
Comment (saruuja) [inline index.html:50]: please don't leave the open tag without close, and close the li element tag in your  Index file

File: style.css [hunk 2]
```diff
@@ -20,12 +20,12 @@ body {
...
 
-.img-div{
+.img-div {
   max-width: 800px;
   margin: 0 auto;
   padding: 15px;
   border: 1px solid green;
-  background-color: #ffffff;
+  background-color: #fff;
 }
```
Comment (saruuja) [inline style.css:30]: please reduce your image width to look your page nice 

[Optional] please make your padding 10px

**Review by saruuja — COMMENTED** [review]
Required Changes ♻️
Hi , This is [@saruuja](https://github.com/saruuja) your code reviewer.
Good job so far!
There are some issues that you still need to work on
Highlights
✔️ Professional Readme
✔️ Good PR title and description
✔️ Meaningful commit messages
✔️ All linters are passing

Suggested changes

please check my comments  below and try to make it 
Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Do not forget to tag me so I am able to assist you as soon as I can 

please

File: index.html (hunk 1 above)
Comment (muneebulrehman) [reply index.html:50]: @saruuja, remember to always be polite. Ok. Instead, phrase it like this. " We should always close the open tags. Don't you think so?"

File: style.css (hunk 2 above)
Comment (muneebulrehman) [reply style.css:30]: When leaving a review, we should make sure, it sounds a suggestion and not an order.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @saruuja  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights


## Required Changes ♻️
_Check the comments under the review._

- You could have raised an issue on the lack of description for the project in the readme file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (saruuja) [inline style.css:30]: let's make your image width small, isn't look that  nice

**Review by Munsa1 — APPROVED** [review]
Hi @saruuja , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions
- [ ] The comments could be more descriptive, by adding a "why" a change is particular 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**